app/models/discovery_rule.rb

```diff
@@ -1,4 +1,6 @@
 class DiscoveryRule < ApplicationRecord
+  audited :associated_with => :hostgroup
+  has_associated_audits
```

Comment:
Does this have any other models associated with it? `has_associated_audits` is needed when some other resource defines its audits with, in this example, `:associated_with => :discovery_rule`

Reply:
Yes, good catch!!
It is not needed here as I have removed audit part from `discovery_attribute_set`.